packages/components/components/elvis-stepper/src/react/utils.ts

```diff
@@ -43,25 +45,47 @@ export const numberShouldBeVisible = (
 };
 
 export const generateStatusMessage = (currentStep: number, steps: StepStates, errorSteps: number[]) => {
-  let statusMessage = `På steg ${currentStep}. `;
+  const lang = useLanguage();
```

Comment:
You cannot use a hook inside a utility-function. React hooks may only be called in components or other hooks, so this should instead be refactored. 
Perhaps as a parameter to the function instead, and the hook called in the component?

Reply:
See the note below here: https://react.dev/learn/reusing-logic-with-custom-hooks#hook-names-always-start-with-use
![image](https://github.com/user-attachments/assets/f2b4510f-48e3-49c0-9dfc-5508bc93a804)

Reply:
I have addressed the issue and removed the directly implemented hook in utils.ts. Instead I implemented the hook in both Stepper-horizontal.tsx and vertical.tsx, and passed the lang variable to the utils file as a parameter.